gitbutler-ui/src/lib/components/Spacer.svelte

```diff
@@ -1,7 +1,21 @@
 <script lang="ts">
+	export let margin: number | undefined = 16;
+
+	const getPadding = () => {
```

Comment:
Named padding but is actually margins. :)

Reply:
it should be `getMargins`. Thank will fix this